modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -817,45 +808,52 @@ class SampleCodeTest {
         }
         // end::replication-status[]
     }
-
-//  BEGIN PendingDocuments IB -- 11/Feb/21 --
+    
+    //  BEGIN PendingDocuments IB -- 11/Feb/21 --
     func dontTestReplicationPendingDocs() throws {
-      // tag::replication-pendingdocuments[]
-
-      let url = URL(string: "ws://localhost:4984/mydatabase")!
-      let target = URLEndpoint(url: url)
-
-      let config = ReplicatorConfiguration(database: database, target: target)
-      config.replicatorType = .push
-
-      // tag::replication-push-pendingdocumentids[]
-      self.replicator = Replicator(config: config)
-      let mydocids:Set = self.replicator.pendingDocumentIds() // <.>
-
-      // end::replication-push-pendingdocumentids[]
-      if(!mydocids.isEmpty) {
-        print("There are \(mydocids.count) documents pending")
-
-        self.replicator.addChangeListener { (change) in
-          print("Replicator activity level is \(change.status.activity.toString())")
-          // iterate and report-on previously
-          // retrieved pending docids 'list'
-          for thisId in mydocids.sorted() {
-            // tag::replication-push-isdocumentpending[]
-            if(!self.replicator.isDocumentPending(thisid)) { // <.>
-              print("Doc ID \(thisId) now pushed")
+        // tag::replication-pendingdocuments[]
+        
+        let url = URL(string: "ws://localhost:4984/mydatabase")!
+        let target = URLEndpoint(url: url)
+        
+        var config = ReplicatorConfiguration(database: database, target: target)
+        config.replicatorType = .push
+        
+        // tag::replication-push-pendingdocumentids[]
+        self.replicator = Replicator(config: config)
+        let mydocids:Set = try self.replicator.pendingDocumentIds() // <.>
+        
+        // end::replication-push-pendingdocumentids[]
+        if(!mydocids.isEmpty) {
+            print("There are \(mydocids.count) documents pending")
+            let thisid = mydocids.first!
+            
+            self.replicator.addChangeListener { (change) in
+                print("Replicator activity level is \(change.status.activity)")
+                // iterate and report-on previously
+                // retrieved pending docids 'list'
+                for thisId in mydocids.sorted() {
```

Comment:
Right, not required. 

I think we also could rethink the example, since checking `isPendingDocumentId:` from changeListener(which can get called every time replicator changes the status is worth demo). It's up to them to use, but our examples shouldn't promote to use in frequently updated sections of code right?